static/espeasy.js

```diff
@@ -1,11 +1,41 @@
 // CodeMirror, copyright (c) by Marijn Haverbeke and others
 // Distributed under an MIT license: https://codemirror.net/LICENSE
+
+var commonAtoms = ["And", "Or"];
+var commonKeywords = ["If", "Else", "Elseif", "Endif"];
+var commonCommands = ["AcessInfo", "Background", "Build", "ClearAccessBlock", "ClearRTCam", "Config", "ControllerDisable",
```

Comment:
Looks like "AcessInfo" has a typo, according to the code it should be "AccessInfo"